script/onlyswaps/json/README.md

```diff
@@ -1,12 +1,14 @@
 ## Contract Deployment Input JSON
 
 To resolve dependencies between the contract deployments, a `.json` file named `<chain-id>.json` is created in this [script/randomness/json](.) folder and populated with contract addresses for the following contracts after they are deployed (either as single deployments or part of the single run deployment for all contracts) to ensure that other dependent scripts can fetch the addresses:
-* Router
+* RouterProxy
+* RouterImplementation
 * RUSD
-* BN254SignatureScheme
+* BN254SwapRequestSignatureScheme
+* BN254ContractUpgradeSignatureScheme
 
-For example, running the following command writes a JSON property `{"bn254SignatureSchemeAddress": "0x7D020A4E3D8795581Ec06E0e57701dDCf7B19EDF"}` to the `<chain-id>.json` file:
+For example, running the following command writes a JSON property `{"bn254SwapRequestSignatureSchemeAddress": "0x7D020A4E3D8795581Ec06E0e57701dDCf7B19EDF"}` to the `<chain-id>.json` file:
```

Comment:
not necessary in this PR, but I wonder if it makes sense to have a separate repo/dependency for our signature scheme